public/js/pimcore/object/tags/objectbricks.js

```diff
@@ -300,17 +300,12 @@ pimcore.object.tags.objectbricks = Class.create(pimcore.object.tags.abstract, {
                     afterrender: function (childConfig, dataProvider, manuallyAdded, panel) {
                         if (!panel.__tabpanel_initialized) {
                             var copy = Ext.decode(Ext.encode(childConfig));
-                            var children = this.getRecursiveLayout(copy, null, {
+                            var children = this.getRecursiveLayout(copy, this.fieldConfig.noteditable, {
                                 containerType: "objectbrick",
                                 containerKey: type,
                                 ownerName: this.fieldConfig.name,
                                 applyDefaults: manuallyAdded
                             }, false, true, dataProvider);
-                            if (this.fieldConfig.noteditable && children) {
-                                children.forEach(function (record) {
-                                    record.disabled = true;
-                                });
-                            }
```

Comment:
to keep the grayed mask, i added
```
if (this.fieldConfig.noteditable){
    panel.mask();
}
```
and a css rule as 

```
.pimcore_objectbrick_item.x-masked .x-mask{
    pointer-events: none;
}
```
to make it work, can you please also verify? 

Then it should go on 10.6 as well. Let me know if you can do that before the end of the day, because i may have to takeover as tomorrow would be the last 10.6 release

Reply:
Follow-up note: this suggestion to add a base `mask` is meant to be only valid for 10.6 to avoid the display changes, in 11 it can be considered to be removed https://github.com/pimcore/pimcore/pull/15759#issue-1854317248

Reply:
the removal of the mask would be postponed a little bit and may take a confirmation first, merging this PR as the equivalent/port of https://github.com/pimcore/pimcore/pull/15759